src/components/regionalization/RegionalizationModal/RegionalizationModal.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect } from 'react'
 import { Modal as UIModal } from '@faststore/ui'
-import PostalCodeInput from 'src/components/common/PostalCode'
+import PostalCodeInput from 'src/components/common/PostalCode/PostalCodeInput'
```

Comment:
```suggestion
import { PostalCodeInput } from 'src/components/common/PostalCode'
```

* make sure to rename the `PostalCode/PostalCodeBar.tsx` function component's name as well.